src/Symfony/Component/Security/Core/Encoder/MessageDigestPasswordEncoder.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Symfony\Component\Security\Core\Encoder;
 
-use Symfony\Component\PasswordHasher\Hasher\MessageDigestPasswordHasher;
```

Comment:
the use statement is still necessary for the deprecation

Reply:
Restored!